### ISSUE TITLE
Create wiki

### ISSUE THREAD

**Issue by breezykermo** [forensic-architecture/timemap#128]
Initially should include:
- link to https://forensic-architecture.org/investigation/timemap-for-cartographic-platforms.
- detailed explanation of the structure of the example sheet, with running FAQ for possible errors. 
- extended details on how to configure options from `config.js`, overriding store.

**Comment by breezykermo** [comment]
Good first issue in the sense that it will require a good understanding of the example sheet, and how different features work. If anyone's interested, send me a note and I'd be happy to help detail.

**Comment by lostground** [comment]
Sorry I can't find a way to message directly...

I am starting to work again on implementing this with some of my own data. I'd be happy to help contribute to the wiki as I do this. What specific sections need work?

**Comment by breezykermo** [comment]
Hiya @lostground, the main additional documentation that would be great would be detailed notes regarding options in the store (whose default values, as noted above, are overridden by setting values in `config.js`). 

For example, which value to set to reconfigure the map anchor, modify the timeline starting date and bounds, etc.

**Comment by breezykermo** [comment]
I rewrote and checked the tutorial linked above about a week ago, so I think that those installation notes are all up to date. I just haven’t got around to writing much in the way of guidance for folks looking to customise instances. Let me know if that makes sense!

**Comment by lostground** [comment]
Yes, that makes sense and I will work on this alongside getting my instance up and running. If I have any questions I'll let you know.

In the meantime I created a small wiki page on how to use 'Sources' which I have been revisiting in the last days. I don't know how it's possible to contribute to an existing wiki on github so I created [my own repository with wiki files](https://github.com/lostground/timemap-wiki) which you can look at. This is my first time contributing with github so if I'm doing something wrong please let me know!

I've also made a [small pull request](https://github.com/forensic-architecture/timemap/pull/142) so that sources in the cards are named according to their Titles rather than their IDs. I imagine this was the expected behaviour so I already included it in how I wrote the 'Sources' wiki page. If it isn't merged I'll change it back.

**Comment by breezykermo** [comment]
This is partly covered by [a blog post on our website](https://forensic-architecture.org/investigation/timemap-for-cartographic-platforms). However getting better documentation in general is something we still need!

**Comment by kdent** [comment]
I'm planning to submit a proposal to Bellingcat's Technical Writing Fellowship. I suggested the idea of contributing to the Wiki for Timemap, which they are interested in. I was hoping  understand more about the priorities for documenting Timemap. Is this still relevant and something I should take up? Thanks!

**Comment by breezykermo** [comment]
Hi @kdent, thanks for your interest here. Timemap isn't very actively maintained at the moment-- [Bellingcat's fork](https://github.com/bellingcat/ukraine-timemap) has a better feature set at this stage, and I haven't done a good job at merging those changes back upstream.

The original idea was that this repo would remain as a generalized application, i.e. as a way to create interactive cartographic platforms in general, whereas the Bellingcat fork is just for the instance documenting the war in Ukraine. Maintaining this repo in line with that philosophy is no longer one of our (Forensic Architecture's) priorities, however, and so it doesn't make sense to keep us this pretense.

What I would suggest as useful work is to document this timemap repository "as-is" in contrast to the Bellingcat fork, such that we can archive this repo. What would make the most sense here, I think, is a basic assessment of how one might create a new timemap instance on the basis of Bellingcat's fork (rather than from this repo), which we can add to the README here in the process of archiving, or as a separate document. Does that make sense? cc. @msramalho who will have better perspective on whether this works in terms of Bellingcat's priorities with the Ukraine platform.

**Comment by kdent** [comment]
Thanks for the context and update. That's very helpful. I'll also confer with the folks managing the fellowship to hear their thoughts on priorities around this.

**Comment by msramalho** [comment]
Thanks for the context @breezykermo. Indeed given the fork divergence Bellingcat's repo has some more features and maintenance updates though it ends-up having specific customizations for Ukraine platform (loosing part of the generalized application goals of this repo). A documentation of Bellingcat's fork with a specific focus on distinguishing what is general (ie reusable for other deployments) from what is Ukraine platform-specific can still be useful for anyone forking from there.
